core/src/replay_stage.rs

```diff
@@ -561,7 +561,7 @@ impl ReplayStage {
             Self::reset_poh_recorder(
                 &my_pubkey,
                 &blockstore,
-                &working_bank,
+                working_bank,
```

Comment:
note: this is the bank that was getting stuck; the arc itself is moved into poh_recorder, so we don't hold it here anymore.